### DISCUSSION TITLE
Help joining tables

### DISCUSSION THREAD

**Discussion by domath2k** [joomlaboat/custom-tables#73] [Q&A] [answered]
Hello. Can someone please help me understand how to left join a table? When I add/update a record in the main table in the front end, I'd like to be able to add a few related child records on same screen/menu selection.

**Comment by joomlaboat** [comment, marked as answer]
Hello Domath2k,

Use the Table Join field type.

Then you can get value from the connected table like this:

Read the selected field processed value from the joined table:

    Field name - Specify the name of the field to get the value.
    Value Processing Parameters - An array of parameters to process the Joined Table field value. (Optional)

Example:

{{ FieldName.get("birthdate",["Y-m-d"]) }}

Read the unprocessed (pure) value of a selected field of the joined table:

    Field name - Specify the name of the field to get the value from.

Example:

{{ FieldName.getvalue("country") }}

Render the value of a joined table using the selected layout.:

    Layout - The layout to render the joined table record.

Example:

{{ FieldName.layout("PersonProfileLayout") }}


Also, you can use the following tags to get values or records from another table:

{{ tables.getvalue }}

{{ tables.getrecord }}

{{ tables.getrecords }}

These methods are available in the Pro version.

**Reply by domath2k** [reply]
Thanks. I'll give it a try
